src/main/resources/db/migration/V75__add_suoritusjako_tehty_to_opiskeluoikeus.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE opiskeluoikeus
+  ADD IF NOT EXISTS suoritusjako_tehty BOOLEAN;
+
+UPDATE opiskeluoikeus
+SET suoritusjako_tehty = false
+WHERE koulutusmuoto IS NULL;
```

Comment:
koulutusmuoto IS NULL?

Reply:
uff, huono copypaste, piti olla WHERE suoritusjako_tehty IS NULL